client/src/api.py

```diff
@@ -57,6 +57,15 @@ def create_app(config: Configuration, tnc_pool: TNCPool, sids_relay: SIDSRelay)
     def get_sids_status():
         return jsonify(sids_relay.get_status()), 200
 
+    @app.route("/api/sids/toggle", methods=["POST"])
+    def toggle_relay():
+        some_json = request.get_json()
+        value = some_json['Mission Control']['relay-enabled']
```

Comment:
What "value"? In my opinion it should be like "current_relay_status" or something like that... Change!